example/gluon/style_transfer/dataset/download_dataset.py

```diff
@@ -26,6 +26,8 @@ def unzip_file(filename, outpath):
         z.extract(name, outpath)
     fh.close()
 
```

Comment:
We need to also put the information below in the corresponding documents. In this case, it would be `example/gluon/style_transfer/README.md`.

Reply:
done